app/jobs/seed_bpn_test_data.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+class SeedBPNTestData < ApplicationJob
+  include Importers::SeedBPNDeclarations
+
+  def perform
+    require "factory_bot"
+
+    ParticipantDeclaration.destroy_all
+    ActiveRecord::Base.transaction do
```

Comment:
why does this whole block need to be in a transaction?

Reply:
much faster